region/maridia/outer.json

```diff
@@ -889,6 +889,24 @@
           "homeNodes": [ 5, 7 ]
         }
       ],
+      "reusableRoomwideNotable": [
+        {
+          "name": "Main Street Crab Climb with Only Ice and Supers",
```

Comment:
This isn't reused, not that it _needs_ to be.

Reply:
Yeah, the second half (mid climb with supers) is in #645